src/oop/special_class_method/medium/q13/Detergent.java

```diff
@@ -0,0 +1,19 @@
+package oop.special_class_method.medium.q13;
+
+public class Detergent {
+    private double volume;
+    private String manufacturer;
+
+    public Detergent(double c, String m) {
```

Comment:
since we'll be giving the constructor to the students the 'c' should probably be switched to a 'v'